packages/core/src/util/error/shared.ts

```diff
@@ -20,7 +20,10 @@ export function genericErrorToString(err: unknown): string {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function detailedErrorToString(err: any): string {
   const thisErrorMessage = err.message;
-  const additionalInfo = err.additionalInfo ? inspect(err.additionalInfo) : undefined;
+  // this can lead to multi-line
+  const additionalInfo = err.additionalInfo
+    ? inspect(err.additionalInfo, { compact: true, breakLength: undefined })
+    : undefined;
```

Comment:
why are we doing this?

Reply:
So we can add additional info to exceptions when using `MetriportException` or its subclasses. This allows us to do so without having to capture errors/msgs to Sentry on multiple levels on the stack.